From 5b3bde1e1da902fc85c07cdd2e2d04d5913b59e2 Mon Sep 17 00:00:00 2001
From: Subrahmanyam Nilla <subrahmanyam.nilla@cavium.com>
Date: Fri, 8 Mar 2019 14:16:14 +0530
Subject: [PATCH 027/386] soc: octeontx2-dpi: disable DPI PF driver by default
 in Kconfig

Change-Id: I80c1a85c91a7ce5dd996c989034bafd4c2c66be6
Signed-off-by: Subrahmanyam Nilla <subrahmanyam.nilla@cavium.com>
Reviewed-on: https://sj1git1.cavium.com/5370
Reviewed-by: Stanislaw Kardach <Stanislaw.Kardach@cavium.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Tested-by: sa_ip-sw-jenkins
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 drivers/soc/marvell/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/soc/marvell/Kconfig b/drivers/soc/marvell/Kconfig
index 3821423ee03d..d96cbc842071 100644
--- a/drivers/soc/marvell/Kconfig
+++ b/drivers/soc/marvell/Kconfig
@@ -70,7 +70,6 @@ config OCTEONTX2_RM_DOM_SYSFS
 config OCTEONTX2_DPI_PF
 	tristate "OcteonTX2 DPI-DMA PF driver"
 	depends on ARM64 && PCI
-	default y
 	---help---
 	  Select this option to enable DPI PF driver support.
 	  DPI (DMA packet interface) provides DMA support for MAC.
-- 
2.17.1

